### DISCUSSION TITLE
current race

### DISCUSSION THREAD

**Discussion by krtffl** [siddharth-tewari/F1-TeamRadio#7] [General]
does this also work with team radios during a race?

**Comment by siddharth-tewari** [comment]
Yes, but I didn't update it since over a year ago, so not sure if it will work properly or not without any errors.

**Comment by krtffl** [comment]
alright awesome, thanks for the reply!

**Reply by siddharth-tewari** [reply]
Just to confirm, did it work?

**Reply by krtffl** [reply]
i tried it with races from current season and yes. haven't had the chance to try live during a race though

**Reply by siddharth-tewari** [reply]
Ok, thank you for telling. 
I think it won't work during a live session as it uses recorded links which are probably published after the race ends.
